tests/data/sending_test_vectors.json

```diff
@@ -1,823 +1,2673 @@
 [
-  {
-    "comment": "Simple send: two inputs",
-    "given": {
-      "outpoints": [
-        [
-          "f4184fc596403b9d638783cf57adfe4c75c605f6356fbc91338530e9831e9e16",
-          0
+    {
```

Comment:
is this directly ported from BIP? no modifications?

Reply:
Yep, no modifications.